src/amber/cli/commands/routes.cr

```diff
@@ -5,9 +5,10 @@ require "../helpers/sentry"
 module Amber::CLI
   class MainCommand < ::Cli::Supercommand
     class Routes < Command
-      RESOURCE_ROUTE_REGEX = /(\w+)\s+\"([^\"]+)\",\s*(\w+)(?:,\s*(\w+)\:\s*\[([^\]]+)\])?/
-      VERB_ROUTE_REGEX     = /(\w+)\s+\"([^\"]+)\",\s*(\w+),\s*:(\w+)/
-      PIPE_SCOPE_REGEX     = /routes\s+\:(\w+)(?:,\s+\"([^\"]+)\")?/
```

Comment:
Did any of these actually change? Or is this a whitespace diff?

Reply:
They changed. Two changes are made:

1) A ":" is added inside the regex so that controllers containing ":" in the name would work properly
2) And () were added around 'routes' and 'websockets' words even though not strictly necessary technically. (I've added them for consistency with other regexes and future expansion to cover multiple words)